docs/dev/building-on-zksync/hello-world.md

```diff
@@ -28,18 +28,22 @@ Please note that breaking changes were introduced in `zksync-web3 ^0.13.0`. The
 
 1. Initialize the project and install the dependencies. Run the following commands in your terminal:
 
-```sh
+::: code-tabs
+
+@tab:active yarn
+
+```bash
+
 mkdir greeter-example
 cd greeter-example
 
-# For Yarn
 yarn init -y
 yarn add -D typescript ts-node ethers@^5.7.2 zksync-web3 hardhat @matterlabs/hardhat-zksync-solc @matterlabs/hardhat-zksync-deploy
 
-# For NPM
 npm init -y
 npm i -D typescript ts-node ethers@^5.7.2 zksync-web3 hardhat @matterlabs/hardhat-zksync-solc @matterlabs/hardhat-zksync-deploy
 ```
+:::
```

Comment:
<img width="913" alt="image" src="https://user-images.githubusercontent.com/72273339/227486080-1647a569-ce39-4e3c-beeb-2d8aa4e505ec.png">

Reply:
in `yarn` tab here is the instruction for npm too. Is has to be changed. npm tab shoud be added

Reply:
@krofax

Reply:
please do as you made it here
<img width="984" alt="image" src="https://user-images.githubusercontent.com/72273339/227494109-830ae3f8-0b71-4aee-8466-869138a8f7ec.png">